src/extensions/scenario/scenarioext_functions.cpp

```diff
@@ -1027,3 +1029,33 @@ void Vinifera_Create_Units(bool official)
     DEBUG_INFO("Finished unit generation. Random number is %d\n", Scen->RandomNumber);
 }
 
+
+/**
+ *  Reimplements a part of scenario [Basic] section reading to support
+ *  reading in new options added to the section.
+ *
+ *  @author: Rampastring
+ */
+void Read_Scenario_Read_Basic_Section_Options(CCINIClass &scenario_ini)
```

Comment:
This patch is not required. Rebase this branch to the upstream develop and add the INI reading line to `ScenarioClassExtension::Read_INI`.